### DISCUSSION TITLE
Avoid Afterburner

### DISCUSSION THREAD

**Discussion by serdarevica** [Lolliedieb/lolMiner-releases#1285] [Q&A]
Hi guys,

I have RTX 3070 cards.

I am using:

`set "EXTRAPARAMETERS=--cclk 1300" ` and it's working fine, 

but as I'm trying to avoid using Afterburner, I would like to be able to overclock memory.
I tried **--mclk**, but it doesn't really help/work in case of mining ETH.

Can you please suggest how to overclock memory?

Thanks!

**Comment by serdarevica** [comment]
I did some extra research and looks like that memory OC will notwork with lolMiner :/

I'm just worried if Windows crash and Afterburner fail to start for some reason.... there must be another way of OC memory that will always work?

Thanks

**Comment by heartoflusho** [comment]
You can overclock the memory in the batch file but it's an absolute value, not an offset.  I use the following for a non-LHR 3070 and get 60mh 1.9gh dual mining eth+ton:

set "EXTRAPARAMETERS=--lhrtune off --cclk 1500 --mclk 8000"

**Reply by serdarevica** [reply]
That is weird!
I did copy and paste your parameters and not getting those clocks on nonlhr 3070s... I have tried on 5 diff ones.
Both cclock and mclck are showing locked in miner, but miner after 3 mins is still showing mem at 6800.
Speeds I'm getting are 51 for ETH 1.6gh for ton

**Reply by heartoflusho** [reply]
You might try lower clocks like 1400 7600, this one is kinda lucky in the silicon lottery.
![image](https://user-images.githubusercontent.com/100519933/155904493-2f6244c0-1668-4c1f-8db5-8ee080e6b089.png)

https://hashrate.no/3070 has some good suggested settings to help guide you though it's using offsets for linux I think.  I had the same issue as you with a zotac 3060 LHR and it took me an hour or two to figure out that I had to keep playing with the memory clock in the batch file until it actually locked it and didnt crash the card.
